wolfcrypt/test/test.c

```diff
@@ -6036,10 +6083,15 @@ WOLFSSL_TEST_SUBROUTINE wc_test_ret_t sm3_test(void)
 WOLFSSL_TEST_SUBROUTINE wc_test_ret_t hash_test(void)
 {
 #if defined(WOLFSSL_SMALL_STACK) && !defined(WOLFSSL_NO_MALLOC)
+    /* The first item in the typesGood array will be used for placeholder */
+    #define          PLACEHOLDER_TYPE 0
+    int              this_type = PLACEHOLDER_TYPE;
```

Comment:
What is this?

Reply:
This seems obvious right? Index 0 is always the first. Is a macro needed? Seems like noisy code